.direnv/flake-profile

```diff
@@ -0,0 +1 @@
+flake-profile-1-link
```

Comment:
Can we get `.direnv` added to the `.gitignore`?

Reply:
d'oh